gradle/base/code-generator-model/src/main/resources/jre.yaml

```diff
@@ -0,0 +1,30 @@
+- version: 8
+- version: 9
+- version: 10
+- version: 11
+- version: 12
+  since: '5.4'
+- version: 13
+  since: '5.4'
+- version: 14
+  since: '5.5'
+- version: 15
+  since: '5.6'
+- version: 16
+  since: '5.7'
+- version: 17
+  since: '5.7.1'
+- version: 18
+  since: '5.8.1'
+- version: 19
+  since: '5.9'
+- version: 20
+  since: '5.9'
+- version: 21
+  since: '5.9.2'
+- version: 22
+  since: '5.10'
+- version: 23
+  since: '5.11'
+- version: 24
+  since: '5.11'
```

Comment:
:speech_balloon: New JDK versions can now be added with two lines. :tada:

Reply:
😱

Reply:
_I recognize a code wizard when I see one._ Well done, Marc! 🧙